user_docs/en/userGuide.t2t

```diff
@@ -795,8 +795,8 @@ You can enable the Screen Curtain in the [vision category #VisionSettings] of th
 
 When the Screen Curtain is active some tasks directly based on what appears on the screen such as performing [OCR #Win10Ocr] or taking a screenshot cannot be achieved.
 
-Screen Curtain is no longer compatible when using versions of NVDA older than 2021.2 with Windows newer than Windows 10 21H2 (10.0.19044).
-For security purposes, confirm that the Screen Curtain works properly when using a new version of Windows for the first time.
+Screen Curtain is no longer compatible when using versions of NVDA older than 2021.2 with Windows 10 21H2 (10.0.19044) or later.
+For security purposes, get visual confirmation that the Screen Curtain works properly when using a new version of Windows for the first time.

```

Comment:
Isn't this issue resolved in 2021.2 via #12684

Reply:
Oh, I misread this. I think this is a little confusing to have in the userguide for 2021.2

Reply:
The inclusion of this comment is something @michaelDCurran suggested - we cannot guarantee this API works on untested versions of Windows.

Reply:
If someone in the future is using 2021.2 with Windows 12, they should not expect the screen curtain to work.

Reply:
I believe that was only necessary in the absence of #12684 which automates the "visual check" and may not be ready to go into 2021.2, there isn't a great deal of time to be confident it works in all cases. Of course, our approach to detecting the screen color could also break in a new version of windows.

Reply:
>Of course, our approach to detecting the screen color could also break in a new version of windows.

Exactly - I don't think we can reliably future proof the screen curtain, and as such we should include a warning.